code-challenges.js

```diff
@@ -47,3 +139,28 @@ const number2 = 100
 // Set two:
 const number3 = 27
 const number4 = 24
+
+// Pseudo Code
+// create a function named subtractLarger
+// subtractLarger will take two arguments (numOne, numTwo) of type Number
+// we will have two more variables store our smaller and larger number (larger, smaller)
+// we can use a conditional compare the two numbers
+// if numOne is greater than or equal to numTwo: assign numOne to larger and numTwo to smaller (this will handle if both numbers are the same)
+// else numTwo is greater and numOne is smaller
+// we can subtract larger from smaller
+
+// Code
+const subtractLarger = (numOne, numTwo) => {
```

Comment:
Alright! Highly impressed with your work on this assessment. Love how you are testing out different methods.